src/docker-in-docker/install.sh

```diff
@@ -561,6 +563,20 @@ dockerd_start="AZURE_DNS_AUTO_DETECTION=${AZURE_DNS_AUTO_DETECTION} DOCKER_DEFAU
         DEFAULT_ADDRESS_POOL="--default-address-pool $DOCKER_DEFAULT_ADDRESS_POOL"
     fi
 
+    if [ -z "$DOCKER_DEFAULT_IP6_TABLES" ]
+    then
+        DEFAULT_IP6_TABLES=true
+    else
+        DEFAULT_IP6_TABLES=$DOCKER_DEFAULT_IP6_TABLES
+    fi
```

Comment:
Why don't we set `DOCKER_DEFAULT_IP6_TABLES` to `true` if the value is not set in the code above? It will avoid this.

Reply:
done 👍